lib/Utils/Backends.pm

```diff
@@ -101,12 +101,12 @@ sub is_remote_backend {
 
 =head2 has_ttys
 
-Returns true if the current instance is using ttys for: ipmi, s390x, spvm, pvm_hmc, except S390_ZKVM
+Returns true if the current instance is using ttys
 
 =cut
 
 sub has_ttys {
-    return ((get_var('BACKEND', '') !~ /ipmi|s390x|spvm|pvm_hmc/) && !get_var('S390_ZKVM'));
+    return ((get_var('BACKEND', '') !~ /ipmi|s390x|spvm|pvm_hmc/) && !get_var('S390_ZKVM') && !(check_var('BACKEND', 'generalhw') && !defined(get_var('GENERAL_HW_VNC_IP'))));
```

Comment:
could you include "generalhw" in the regex-check? I.e. `… !~ /ipmi|s390x|spvm|pvm_hmc|generalhw/ …`

Reply:
It will not be exactly the same condition. 
For example, if you have `BACKEND=qemu` but `GENERAL_HW_VNC_IP` defined, because you have cloned an existing job, and `GENERAL_HW_VNC_IP` was defined in previous job and is a leftover in the new one.